src/MembershipsSDKAdapter.js

```diff
@@ -31,24 +31,44 @@ const MAX_MEMBERSHIPS = 1000;
  * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/MembershipsAdapter.js#L6}
  */
 
+/**
+ * Sort the members alphabetically, with the current user first
+ *
+ * @param {Array} members List of sdk meeting members
+ * @returns {Array} Sorted list of sdk meeting members
+ */
+function sortMeetingMembers(members) {
+  return members.sort((member1, member2) => (
+  /* eslint-disable no-nested-ternary */
+    member1.isSelf ? -1// current user comes first
```

Comment:
please add a single space before the comment

Reply:
could you also disable the indentation rule that prevents the vertical aligning of these conditions?